src/Control/Monad/Morph.hs

```diff
@@ -116,6 +120,54 @@ class MFunctor t where
     -}
     hoist :: (Monad m) => (forall a . m a -> n a) -> t m b -> t n b
 
+{-| Convert a Monad to an isomorphic transformation of Identity
```

Comment:
Instead of "isomorphic transformation of Identity" we could say "equivalent monad transformer"

Reply:
I thought of that wording first and discarded because they're not required to be equivalent. Probably my current wording is worse though.

Reply:
I view the words "isomorphic" and "equivalent" as interchangeable (although isomorphic is technically more precise).  The only reason I suggest "equivalent" is that it might be less intimidating to some people

Reply:
Maybe Convert a Monad to an equal transformer stack. I was just learning some HoTT and equal means isomorphic in HoTT - so how about that?

Reply:
Yeah, "equal" would be fine, too